arviz/stats/diagnostics.py

```diff
@@ -530,6 +535,25 @@ def _bfmi(energy):
     return num / den
 
 
+def _backtransform_ranks(arr, c=3 / 8):
+    """Backtransformation of ranks.
+    
+    Parameters
+    ----------
+    arr : np.ndarray
+        Ranks array
+    c : float
+        Fractional offset. Defaults to c = 3/8 as recommended by Bloom (1985).
```

Comment:
@avehtari any idea what paper is this referring to?

Reply:
There are two typos in the reference. It should be Blom (1958). The reference can be found in the Rhat paper

Blom, G. (1958). Statistical Estimates and Transformed Beta-Variables. Wiley; New York.

Reply:
Thanks